### DISCUSSION TITLE
"Path-based and triplification approaches to mapping data into RDF: user behaviours and recommendations" accepted for publication in the SWJ

### DISCUSSION THREAD

**Discussion by luigi-asprino** [SPARQL-Anything/sparql.anything#509] [Announcements]
[This](https://www.semantic-web-journal.net/content/path-based-and-triplification-approaches-mapping-data-rdf-user-behaviours-and-0) is the link to the preview of the article.

I look forward to reading your comments on the work :-)

**Comment by justin2004** [comment]
" 11.2. Future developments for YARRRML and
SPARQL Anything "
... with respect to SPARQL Anything
"Another approach would be to display part
of the triplification, as required by the user, or to
provide standard queries to interrogate the triplification"

when i was first learning sparql anything my first query was always something like:
```
PREFIX  xyz:  <http://sparql.xyz/facade-x/data/>
PREFIX  fx:   <http://sparql.xyz/facade-x/ns/>
CONSTRUCT { ?s ?p ?o  }
WHERE
  { SERVICE <x-sparql-anything:>
      { fx:properties
                  fx:location     "some.csv" ;
                  fx:csv.headers  "true" .
        ?s        ?p              ?o
      }
  }
```

and then i would keep that output open in a pane and then i would imagine that all my additional querying was targeting it. in that way, i was only thinking about vanilla SPARQL queries... sparql anything was out of the way.

**Reply by luigi-asprino** [reply]
That's true. ?s ?p ?o pattern works for any RDF dataset and it a good starting point for exploring an unknown graph.
By the way, it is one of the first queries suggested in the tutorial #508
